roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/ds-ingress-nginx-controller.yml.j2

```diff
@@ -29,6 +29,7 @@ spec:
       nodeSelector:
         {{ ingress_nginx_nodeselector | to_nice_yaml }}
 {%- endif %}
+        node-role.kubernetes.io/ingress: "true"
```

Comment:
?

Reply:
I think it came from some other place (I had to do big rebase and resolve many conflicts), will remove if.